tools/roosttest/test_osc_pipeline.py

```
@@ -152,18 +157,17 @@ def test_inverse_resolver_call_site_swaps_fg_bg(self, roost, project):
     def test_osc11_set_then_query_replies_with_new_bg(self, roost, project):
-        """Pre-fix the OSC drain read the static theme bg, so a
-        mid-session `OSC 11;rgb:00/11/22` set would NOT be reflected
-        in the next `OSC 11;?` query reply. Post-fix (#145) it reads
-        libghostty's live colors. SET in one feed, QUERY in a second
-        — libghostty processes SET via vt_write before the QUERY's
-        scanner.feed runs, so the reply uses the post-set bg."""
+        """A mid-session `OSC 11;rgb:00/11/22` set must be reflected in
+        the next `OSC 11;?` reply: libghostty answers from
+        `override orelse default`, and the set moved the override. SET
+        in one feed, QUERY in a second."""
         tab = roost.open_tab(project, cwd="/tmp")
```

Ruff (0.16.2)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)